### PR TITLE
Updating the name of the AR bug template

### PR DESCRIPTION
**Pull request by mcphedar**
Signed-off-by: mcphedar <mcphedar@amazon.com>

### CODE REVIEW

**Review by tjmichaels — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/ar_bug_report.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: ar_bug_report.md
+name: Automated_Review_bug_template.md
```
Comment (hultonha) [inline .github/ISSUE_TEMPLATE/ar_bug_report.md:2]: suggestion: Could this be `Automated Review bug template` instead (no underscores or `.md` extension)

**Review by hultonha — COMMENTED** [review]
Small bit of feedback

File: .github/ISSUE_TEMPLATE/ar_bug_report.md (hunk 1 above)
Comment (mcphedar) [reply .github/ISSUE_TEMPLATE/ar_bug_report.md:2]: Good catch, Tom.

**Review by hultonha — APPROVED** [review]
Awesome thanks!

**Review by tjmichaels — APPROVED** [review]

**Pull request merged**